src/library/general_library.lua

```diff
@@ -526,5 +526,37 @@ function library.calc_script_name(include_extension)
     return retval
 end
 
+--[[
+% get_used_font_name
+
```

Comment:
I am unclear why this function is needed. Can you give an example of an issue it addresses? Also, I think I a clearer name would be `calc_font_os_name` and I think it should take `FCFontInfo` as its input parameter.

Reply:
This is code that I am abstracting from JetStream that CJ wrote initially. In looking at it, it seems to me like he is calling up a temporary file and analyzing its path to see whether it uses / or \ as a delimiter to determine OS... Does that seem right? Wouldn't it be better to use UI:IsMac() or whatever it is to determine the OS?

In any case, you may be right and this particular step may be unnecessary. I just confirmed that if I set a document to Engraver Font Set on my Mac, fontinfo:GetName() returns "Engraver Font Set", and then when I open the same file in Windows I get "EngraverFontSet". In other words, the transformation that this function does where it strips out spaces in the font name isn't really needed.

I think that I may test this on my JetStream setup without this step just because I'm a little reluctant to change it when it's already working... But if we're right I'll remove this entire function and just go with calc_default_music_font_name() as a single function.

Reply:
If it's just converting from FCString to lua string, I'd say the prefix can stay `get_`.

FWIW: believe it or not, Windows works equally well with the `/` (forward slash). I only learned this within the last 12 months, despite having worked with it for almost 40 years going back to when it was only DOS. All this code to use backslash on Windows and forward slash on Unix/macOS is completely unnecessary.

Reply:
P.S. Also this would ideally be a `FCMFontInfo` mixin, but I'd say we can save that for another day.